src/AddTodoForm.jsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
 
-const AddTodoForm = (props) => {
+const AddTodoForm = ({ onAddTodo }) => {
+  const [todoTitle, setTodoTitle] = React.useState('');
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
 
   const handleAddTodo = (event) => {
     event.preventDefault();
-    const todoTitle = event.target.title.value;
-    props.onAddTodo(todoTitle);
-    console.log(todoTitle);
-    event.target.reset();
+    if (todoTitle.trim()) {
```

Comment:
Good usage of trim method to avoid adding empty todos.

Reply:
Thank you @gsvidal !